packages/astro/src/core/app/index.ts

```diff
@@ -175,7 +175,7 @@ export class App {
 			// https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Forwarded-Proto
 			let protocol = request.headers.get('X-Forwarded-Proto');
 			if (protocol) {
-				// this header doesn't have the colum at the end, so we added to be in line with URL#protocol, which has it
+				// this header doesn't have the column at the end, so we added to be in line with URL#protocol, which has it
```

Comment:
```suggestion
				// this header doesn't have the colon at the end, so we added to be in line with URL#protocol, which has it
```

I think this means colon (`:`)